src/codeu/chat/client/commandline/Chat.java

```diff
@@ -181,6 +185,23 @@ private UserContext findUser(String name) {
       }
     });
 
+    // Added by Priyanka Agarwal according to CodeU Tutorial
+    // Added import statement
+    panel.register("info", new Panel.Command() {
+      @Override
+      public void invoke(Scanner args) {
+        final ServerInfo info = context.getInfo();
+        if (info == null) {
+          // Communicate error to user - the server did not send us a valid
```

Comment:
Minor note: try to comment *after* you've changed something, that way the commenter knows when to come back and look at your code. It's hard to know when to come back if you say you *will* change it. It's a lot easier if you say "I fixed that, please take another look" instead.